src/locales/en.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Sorry, but these are not the instructions i was looking for. i should have referred to `InstructionPage.vue` specifically in the Issue. 

We do not use FormInput or FormResults in Quiz Engine anymore.